backend/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:buster
+
+RUN apt update
+RUN echo yes | apt upgrade
+
+RUN echo yes | apt install nodejs npm curl git htop

```

Comment:
why not use apt-get --yes instead of this sorcery ?

Reply:
As You want, was just send "yes" to the standard Input